### ISSUE TITLE
Add term tibiotalar synostosis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10440]
**Preferred term label:**
Tibiotalar synostosis

**Synonyms**
Talocrural joint synostosis. Tibiotalar joint fusion

**Definition (free text, please give PubMed ID)**
Fusion of the junction between the distal tibia, fibula and the talus. PMID: 27594929

**Parent term (use hpo.jax.org/app)**
Synostosis involving the fibula HP:0005928
Synostosis involving the tibia HP:0005929
Synostosis involving bones of the feet HP:0009140

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Acromesomelic dysplasia, Grebe type (OMIM601146)  PMID27577507

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6001052
- label: Tibiotalar synostosis
- parents: [Synostosis involving bones of the feet (HP:0009140)](https://hpo.jax.org/app/browse/term/HP:0009140) | [Synostosis involving the fibula (HP:0005928)](https://hpo.jax.org/app/browse/term/HP:0005928) | [Synostosis involving the tibia (HP:0005929)](https://hpo.jax.org/app/browse/term/HP:0005929)
- pmid: PMID:27594929|PMID:27577507
- def: Fusion of the junction between the distal tibia, fibula and the talus.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10440
- synonyms:
